.github/workflows/release.yml

```diff
@@ -80,11 +82,15 @@ jobs:
 
       - name: Publish
         run: pnpm --filter ${{ matrix.package }} publish --access public --no-git-checks --ignore-scripts
```

Comment:
According to the link in the PR description, we also need to add the `--provenance` flag here

Reply:
We use pnpm, the link is only describing how to do it with non, pnpm uses the env instead of the argument.

Reply:
Ah, that makes sense. I thought pnpm would proxy the flags.